geoportal/c2cgeoportal_geoportal/views/i18n.py

```diff
@@ -26,23 +26,85 @@
 # either expressed or implied, of the FreeBSD Project.
 
 
+import glob
+import logging
+
 import pyramid.request
 import pyramid.response
-from pyramid.httpexceptions import HTTPFound
+from lingua.extract import (
+    ExtractorOptions,
+    POEntry,
+    create_catalog,
+    find_file,
+    list_files,
+    no_duplicates,
+    read_config,
+    strip_linenumbers,
+)
+from lingua.extractors import get_extractor, register_extractors
+from lingua.extractors.babel import register_babel_plugins
+from pyramid.httpexceptions import HTTPInternalServerError
 from pyramid.view import view_config
 
-from c2cgeoportal_geoportal.lib.cacheversion import get_cache_version
+from c2cgeoportal_geoportal.lib import lingua_extractor
 from c2cgeoportal_geoportal.lib.caching import Cache, set_common_headers
 
+LOG = logging.getLogger(__name__)
+
+
+@view_config(route_name="localepot")  # type: ignore
+def localepot(request: pyramid.request.Request) -> pyramid.response.Response:
+    """Get the pot from an HTTP request."""
+    # Build the list of files to be process
+    package = request.registry.settings["package"]
+    sources = ["/etc/geomapfish/config.yaml", "/app/development.ini"]
+    sources += glob.glob(f"geoportal/{package}_geoportal/static-ngeo/js/apps/**/*.html.ejs")
+    sources += glob.glob(f"geoportal/{package}_geoportal/static-ngeo/js/**/*.js")
+    sources += glob.glob(f"geoportal/{package}_geoportal/static-ngeo/js/**/*.html")
+    sources += glob.glob("/usr/local/tomcat/webapps/ROOT/**/config.yaml")
```

Comment:
Should use only absolute paths (using pkg_resources if needed) ?
Are all these files in runtime container ?

Reply:
moved => no more needed :-)